gating/check/run

```diff
@@ -145,11 +145,35 @@ function _get_data_disk {
   fi
 }
 
+function _download_dependent_components {
+  # Install python3 for Ubuntu 16.04 and CentOS 7
+  if which apt-get; then
+    sudo apt-get install -y python3-virtualenv
+    virtualenv --python python3 .rpc_component_venv
+  fi
+
+  if which yum; then
+    sudo yum install -y python34-virtualenv.noarch
+    virtualenv-3 .rpc_component_venv
+  fi
+
+  set +u; . .rpc_component_venv/bin/activate; set -u
+  pip install "git+https://github.com/rcbops/rpc_component@85f152ece9eee3c9cd7d68bd2ecc5982e17400ae#egg=rpc_component"
+
+  component dependency download-requirements --download-dir /etc/ansible/ceph_roles/
```

Comment:
This is kind of messy -- I wanted to use a `--download-dir` of `/opt` since not all requirements may be ansible roles, but there's some assumption that we put this here as `tests/ansible-role-test-requirements.yml` installs openstack-ansible-tests to /etc/ansible/ceph_roles/rpc-ceph/tests/common.

On a side note, `gating/check/run` also clones `openstack-ansible-tests` to `tests/common` in its own directory. Do we need to clone this twice?